cranelift/codegen/src/isa/zkasm/abi.rs

```diff
@@ -330,13 +337,14 @@ impl ABIMachineSpec for ZkAsmMachineDeps {
         // Compute linkage frame size.
         let setup_area_size = if flags.preserve_frame_pointers()
             || !is_leaf
+            // FIXME: we don’t currently maintain a frame pointer?
```

Comment:
As I know we don't. Also, as I know zkAsm don't have special register for it. ``CTX`` looks like something we want to use here, but I'm not shure.

Reply:
CTX is probably better used for the WASM context (since that's what it is meant for.) Frame pointers aren’t required to exist, the FIXME was largely added here pointing out that at least some of the conditions here are not useful.